### PR TITLE
Add details about project

### PR DESCRIPTION
**Pull request by AaruBama**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,87 @@
...
+art of memes and engage with a diverse community of meme enthusiasts.
+
+Whether you're looking for the latest trends, the quirkiest humor, or simply a
+good laugh, Memestr is your go-to destination. Come on board and let's meme
+together!
+
+## What sets it apart from similar projects?
+
+Memestr sets itself apart from other similar websites like 9gag in several
+unique ways:
```
Comment (nemanjaglumac) [inline README.md:21]: What sets it apart from numerous other Nostr (web) clients that are already well established in the community?

Comment (AaruBama) [reply README.md:24]: Are there any established Nostr clients for memes? I'm not sure about any.

Comment (AaruBama) [reply README.md:24]: In future, memestr will have its own hosted relays with a unique algorithm for providing personalised feed based on data accumulated and processed from the relays but as of now, Memestr is a community for members who like seeing all their memes/news/random stuff at one place without having to go through a lot of text statuses.

Should we be putting that in readme?

Comment (nemanjaglumac) [reply README.md:24]: > Memestr is a community for members who like seeing all their memes/news/random stuff at one place without having to go through a lot of text statuses.

That summary seems great.
I'd suggest keeping readme fairly short and to the point. People should understand what the project is about from a few sentences.

You can link to a more detailed, or a more technical document (for longer explanations) for anyone curious.

Comment (AaruBama) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+Try Now 🎊 https://memestr.app
+
+## What is Memestr?
+
+Memestr is a community where humour takes centre stage. As
+an
+open-source platform, Memestr empowers users to express themselves through the
+art of memes and engage with a diverse community of meme enthusiasts.
+
+## What sets it apart from similar projects?
```
Comment (abhay-raizada) [inline README.md:15]: should also add that memestr is a media only client, @nemanjaglumac what do you think?

Comment (AaruBama) [reply README.md:15]: Would update.

**Pull request merged**